scipy/signal/signaltools.py

```diff
@@ -1038,18 +1087,24 @@ def choose_conv_method(in1, in2, mode='full', measure=False):
 
     Notes
     -----
-    For large n, ``measure=False`` is accurate and can quickly determine the
-    fastest method to perform the convolution.  However, this is not as
-    accurate for small n (when any dimension in the input or output is small).
-
-    In practice, we found that this function estimates the faster method up to
-    a multiplicative factor of 5 (i.e., the estimated method is *at most* 5
-    times slower than the fastest method). The estimation values were tuned on
-    an early 2015 MacBook Pro with 8GB RAM but we found that the prediction
-    held *fairly* accurately across different machines.
-
-    If ``measure=True``, time the convolutions. Because this function uses
-    `fftconvolve`, an error will be thrown if it does not support the inputs.
+    Generally, this method is about 90% accurate (99% for 2D signals and 85%
+    accurate for 2D signals for randomly chosen input sizes). For precision
+    (e.g., for different hardware), use ``measure=True`` to find the fastest
+    method by timing the convolution.
+
+    If this funciton is incorrect, the estimated method will very like by less
+    than 5 times slower than the faster method (at least in our
+    experiments). There is a 99.9% chance of this ratio being less than 2 for 2D
+    signals and less than 5 for 1D signals. This function is most inaccurate for
+    fast 1D convolutions taking less that 0.01 seconds. The speed tests were
```

Comment:
Also would be good to give a rough corresponding transform length / number of samples on the given testing machine
```suggestion
    fast 1D convolutions taking less than 0.01 seconds. The speed tests were
```